### PR TITLE
[CP] Katello 3.18.5

### PR DESCRIPTION
**Pull request by jturel**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+## Bug Fixes
+
+### Repositories
+ * Pulp migration failed with "NoMethodError: undefined method `pulp_id' for nil:NilClass" ([#33278](https://projects.theforeman.org/issues/33278))
+ * Pulp 2 doesn't escape proxy usernames or passwords ([#33220](https://projects.theforeman.org/issues/33220))
+ * The Pulp 3 migration rake task should warn the user that "Migrating content to Pulp 3 ..." may take a long time to complete ([#33207](https://projects.theforeman.org/issues/33207))
+ * foreman-maintain failed to cancel or resume the task before the upgrade. ([#33206](https://projects.theforeman.org/issues/33206))
+ * [pulp3] 2to3 migration progress reporting will be stuck at  "Migrating content to Pulp 3" for a long time ([#33192](https://projects.theforeman.org/issues/33192))
+ * '[pulp3] foreman-rake katello:approve_corrupted_migration_content command failed with rake aborted!' ([#33173](https://projects.theforeman.org/issues/33173))
+ * undefined method `pulp3_remote_href' for nil:NilClass (NoMethodError) when migrating during content view publish ([#33150](https://projects.theforeman.org/issues/33150))
```
Comment (jeremylenz) [inline CHANGELOG.md:12]: The backtick here breaks the rest of the markdown :(

**Review by jeremylenz — COMMENTED** [review]
LGTM, one comment below!

**Comment by jturel** [comment]
@jeremylenz updated!

**Review by jeremylenz — APPROVED** [review]
APJ :+1:

**Pull request merged**